config_examples/config_train.yaml

```diff
@@ -46,24 +45,36 @@ exp_params:
     shape: [100, 200, 200]
   training_data_path: "/path/training/data"
   validation_data_path: "/path/validation/data"
+  # debug_path: "/path/to/save/intermediate/pred/during/training" # remove this if not needed
+
   # hyper parameters
   LR: 0.001
   weight_decay: 0.0005
-  scheduler: 
-    name: 'ExponentialLR' 
-    gamma: 0.93 # 0.935
-    # name: CosineAnnealingLR 
-    # T_max: 40
 
+  # more about scheduler and parameters. https://pytorch.org/docs/stable/optim.html#how-to-adjust-learning-rate
+  scheduler_name: 'ExponentialLR' 
+  scheduler_params: 
+    gamma: 0.95
```

Comment:
Are there two gamma parameters for the scheduler? Thought you defined that in line 54 as a hyper parameter.